packages/react-native-win32-tester/src/js/examples-win32/APIs/FocusManagerAPI.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { FocusManager } from '../../../../../react-native-win32/Libraries/Utilities/FocusManager.win32';
```

Comment:
For 0.64+, we will want to import from “@office-iss/react-native-win32”.

Relative import should be for 0.62/0.63 where this was inside the same package, and importing the package from inside the package caused a circular dependency.